lambda/event.json

```diff
@@ -0,0 +1,209 @@
+            },
+            "terminateInstanceOnFailure": true
+          },
+          "integrationJobId": "16ca9f96-671d-4a32-8191-cfaa0945e152",
+          "osVersion": "Amazon Linux 2",
+          "outputResources": {
+            "amis": [
+              {
+                "name": "rhythmic_amzn2_base_ 2020-06-22T20-59-00.506Z",
+                "image": "ami-0dd050facfbdb3127",
+                "region": "us-east-1"
+              },
+              {
+                "name": "rhythmic_amzn2_base_ 2020-06-22T20-59-00.506Z",
+                "image": "ami-04115edc30dfe5b06",
+                "region": "us-east-2"
+              },
+              {
+                "name": "rhythmic_amzn2_base_ 2020-06-22T20-59-00.506Z",
+                "image": "ami-016892d4034d0f746",
+                "region": "us-west-1"
+              },
+              {
+                "name": "rhythmic_amzn2_base_ 2020-06-22T20-59-00.506Z",
+                "image": "ami-00999658d4b25441d",
+                "region": "us-west-2"
+              },
+              {
+                "name": "rhythmic_amzn2_base_ 2020-06-22T20-59-00.506Z",
+                "image": "ami-0c7b44c0412b92110",
+                "region": "ca-central-1"
+              }
+            ]
+          },
+          "platform": "Linux",
+          "semver": 1237940040438301781653454858,
+          "sourcePipelineArn": "arn:aws:imagebuilder:us-east-1:312594956781:image-pipeline/rhythmic-amzn-2-base",
+          "state": {
+            "status": "AVAILABLE"
+          },
+          "tags": {
+            "internalId": "39e86d24-e652-408f-829e-e46f8459f8c5",
+            "resourceArn": "arn:aws:imagebuilder:us-east-1:312594956781:image/amazon-linux-base-image-recipe-50b5a411a6b5/1.1.2/10"
+          },
+          "testExecutionId": "3ece22e3-6721-4b62-9ae1-0749ffa37fed",
+          "version": "1.1.2",
+          "versionlessArn": "arn:aws:imagebuilder:us-east-1:312594956781:image/amazon-linux-base-image-recipe-50b5a411a6b5"
+        },
+        "MessageAttributes": {
+          "Test": {
+            "Type": "String",
+            "Value": "TestString"
+          },
+          "TestBinary": {
+            "Type": "Binary",
+            "Value": "TestBinary"
+          }
+        },
+        "MessageId": "95df01b4-ee98-5cb9-9903-4c221d41eb5e",
+        "Signature": "tcc6faL2yUC6dgZdmrwh1Y4cGa/ebXEkAi6RibDsvpi+tE/1+82j...65r==",
+        "SignatureVersion": "1",
+        "SigningCertUrl": "https://sns.us-east-2.amazonaws.com/SimpleNotificationService-ac565b8b1a6c5d002d285f9598aa1d9b.pem",
+        "Subject": "TestInvoke",
+        "Timestamp": "2019-01-02T12:45:07.000Z",
+        "TopicArn": "arn:aws:sns:us-east-2:123456789012:sns-lambda",
+        "Type": "Notification",
+        "UnsubscribeUrl": "https://sns.us-east-2.amazonaws.com/?Action=Unsubscribe&amp;SubscriptionArn=arn:aws:sns:us-east-2:123456789012:test-lambda:21be56ed-a058-49f5-8c98-aedd2564c486"
+      }
+    }
+  ]
```

Comment:
I think a lot of what's in here should be exposed via the Terraform module so that people can supply their own events but I'm not sure what I'm looking at

Reply:
You're looking at trash I should have deleted. Extra copypasta from the first time I used node-lambda